clang/include/clang/AST/Type.h

```diff
@@ -3343,6 +3363,8 @@ class CountAttributedType final
   static bool classof(const Type *T) {
     return T->getTypeClass() == CountAttributed;
   }
+
+  StringRef GetAttributeName(bool WithMacroPrefix) const;
```

Comment:
```suggestion
  StringRef getAttributeName(bool WithMacroPrefix) const;
```

Also a little concerning how specialized this is, rather than being part of how this type is printed.

Reply:
I filed #113585 to track fixing this properly. I don't see a way of fixing this until we have a good way to get the TypeLoc for the attribute (#113582).